src/services/api.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+// TODO config for url
+const baseURL = 'http://localhost:4040/api/users'
+
+export const telescoperApi = {
+  api: axios.create({
+    baseURL,
+  }),
+  setToken(token?: string) {
+    let newVal
+    if (token) {
+      newVal = `Bearer ${token}`
+    } else {
+      newVal = undefined
+    }
+    this.api.defaults.headers.Authorization = newVal
+  },
+  async signUp(email: string, username: string, password: string): Promise<{ username: string }> {
+    const response = await this.api.post('/api/users/', { email, username, password })
```

Comment:
Grant, you dummy, you copied and pasted wrong here
Its requesting from api/users/api/users

Reply:
lol! I'll fix in a sec